odoo/addons/base/models/res_company.py

```diff
@@ -223,7 +224,8 @@ def install_l10n_modules(self):
         uninstalled_modules = self.uninstalled_l10n_module_ids
         is_ready_and_not_test = (
             not tools.config['test_enable']
-            and (self.env.registry.ready or not self.env.registry._init)
+            and (self.env.registry.ready or not self.env.registry._init) and
+            not getattr(threading.current_thread(), 'testing', False)
```

Comment:
```suggestion
            and (self.env.registry.ready or not self.env.registry._init)
            and not getattr(threading.current_thread(), 'testing', False)
```

https://peps.python.org/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

> For decades the recommended style was to break after binary operators. But this can hurt readability in two ways: the operators tend to get scattered across different columns on the screen, and each operator is moved away from its operand and onto the previous line